source/docs/software/wpilib-tools/robot-characterization/analyzing-feedback.rst

```diff
@@ -47,6 +34,8 @@ To specify the correct settings for your PID controller, use the following optio
 - **Has Slave:** Whether there is a motor controller slaved to the controller running the control loop, if the control loop is being run on a peripheral device. This changes the effective loop period.
```

Comment:
update terminology

Reply:
I updated [analyzing-feedback](https://github.com/wpilibsuite/frc-docs/blob/620dac216667bd4195ac993a11b5a71110aa9659/source/docs/software/wpilib-tools/robot-characterization/analyzing-feedback.rst). Are there any more instances of slave in other files?

Reply:
I don't see any